src/Spatial.Roslyn.Tests/UnitVector3dTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace Spatial.Roslyn.Tests
+{
+    using Gu.Roslyn.Asserts;
+    using NUnit.Framework;
+    using SpatialAnalyzers;
+
+    public class UnitVector3dTests
+    {
+        private static readonly ExpectedDiagnostic CS0618 = ExpectedDiagnostic.Create("CS0618");
+
+        [Test]
+        public void ReplaceCtorWithCreate()
+        {
+            var testCode = @"
+namespace RoslynSandbox
+{
+    using MathNet.Spatial.Euclidean;
+
+    class Foo
+    {
+        public Foo()
+        {
+            var uv = new UnitVector3D(1, 2, 3);
+        }
+    }
+}";
+            var fixedCode = @"
+namespace RoslynSandbox
+{
+    using MathNet.Spatial.Euclidean;
+
+    class Foo
+    {
+        public Foo()
+        {
+            var uv = UnitVector3D.Create(1, 2, 3);
+        }
+    }
+}";
+            AnalyzerAssert.CodeFix<UpdateCodeFix>(CS0618, testCode, fixedCode);
```

Comment:
This is how we will write the tests, code before & code after. If you are curious about the assert sources are here: https://github.com/JohanLarsson/Gu.Roslyn.Asserts